obj_spawner.lua

```diff
@@ -0,0 +1,46 @@
+Class = require 'lib.class'
+Vector = require 'lib.vector'
+Timer = require 'lib.timer'
+Coin = require 'coin'
+
+ObjSpawner = Class {
+	init = function(self, objType, spawnInterval, qty)
```

Comment:
we probably don't need these to vary with a constructor (unless we want multiple obj spawner instances)